doc/source/_toc.yml

```diff
@@ -379,26 +380,39 @@ parts:
               sections:
               - file: cluster/vms/examples/ml-example
             - file: cluster/vms/references/index
+          - file: cluster/configure-manage-dashboard
           - file: cluster/running-applications/index
             title: Applications Guide
           - file: cluster/faq
           - file: cluster/package-overview
 
-      - file: ray-observability/monitoring-debugging/monitoring-debugging
+      - file: ray-observability/index
         title: "Monitoring and Debugging"
         sections:
+          - file: ray-observability/getting-started
+          - file: ray-observability/key-concepts
           - file: ray-observability/user-guides/index
             title: User Guides
             sections:
               - file: ray-observability/user-guides/troubleshoot-apps/index
                 title: Troubleshooting Applications
                 sections:
+                  - file: ray-observability/user-guides/troubleshoot-apps/general-troubleshoot
```

Comment:
Shall we name this page "Other common issues" and move it under "optimize-performance"? The reason is that the issues in this page are not the most common ones (I personally have never helped people with those issues).